src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -36,75 +22,88 @@ const ApplicationModal = ({
   const handleCancel = () => {
     setDisplayModal(false);
     setProfileId('');
-    setNotesValue(notes);
-    setHideForm(true);
   };
 
-  const displayForm = () => {
-    setHideForm(false);
-  };
+  const openNotificationWithIcon = (type, status, err) => {
+    if (type === 'success') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User has been approved successfully',
+        });
+      } else {
+        notification[type]({
+          message: 'User has been rejected successfully',
+        });
+      }
+    }
 
-  const handleChange = e => {
-    setNotesValue({
-      ...notesValue,
-      [e.target.name]: e.target.value,
-    });
+    if (type === 'error') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User could not be approved at this time',
+          description: `Error: ${err}`,
+        });
+      } else {
+        notification[type]({
+          message: 'User could not be rejected at this time',
+          description: `Error: ${err}`,
+        });
+      }
+    }
   };
 
-  const addNote = e => {
-    axiosWithAuth()
-      .put(
-        `/application/update-notes/${currentApplication.application_id}`,
-        notesValue
-      )
-      .then(res => {
-        updateModal();
-      })
-      .catch(err => {
-        console.log(err);
-      });
-    e.preventDefault();
-    setHideForm(true);
+  const approveRejectHelper = status => {
+    if (currentApplication.role_name === 'mentor') {
+      return {
+        validate_status: status,
+        current_company: currentApplication.current_company,
+      };
+    } else {
+      return {
+        validate_status: status,
+      };
+    }
   };
 
   /**
-   * Author: Khaleel Musleh
-   * @param {approveApplication} e is for approving an application of a mentor_intake or mentee_intake Boolean from false to approved:true making a PUT call to the backend database server.
+   * Author: John Ruddock
+   * @param {approveApplication} () is for approving an application of a mentor or mentee by altering a validate_status string to approved, making a POST call to the backend database server.
    */
-  /**
-   * Author: Christwide Oscar
-   * @param {onConfirm} e was not created for the application approve and reject buttons. I changed the functions for the onConfirm to onClick and everything seem to work correctly from the console side.
-   */
-  const approveApplication = e => {
+
+  const approveApplication = () => {
     axiosWithAuth()
-      .post(`/application/approve/${currentApplication.profile_id}`, {
-        profile_id: currentApplication.profile_id,
-        low_income: currentApplication.low_income,
-      })
+      .post(
```

Comment:
This bit of code could be a bit more DRY. Try using a conditional variable that replaces the spots where 'approve' or 'reject' are used. The two functions (approveApplication & rejectApplication) are the same otherwise and therefore doesn't need to be repeated